### PR TITLE
Adding Sub(x) prod endpoint

### PR DESCRIPTION
**Pull request by manasiSantFT**
Sub(x) production endpoint has been changed to - https://produk.zeddit.com/www.ft.com/SendEventData

### CODE REVIEW

**Review by ivomurrell — APPROVED** [review]

File: lib/metrics/services.js [hunk 1]
```diff
@@ -308,7 +308,8 @@
 	'subs-graphql-test': /^https:\/\/(beta-)?api-t\.ft\.com\/subs\/query(\/graphql)?(\/)?$/,
 	'subs-graphql-api': /^https:\/\/(beta-)?api\.ft\.com\/subs\/query\/api\//,
 	'subs-graphql-api-test': /^https:\/\/(beta-)?api-t\.ft\.com\/subs\/query\/api\//,
-	'subx-send-conversion-data': /^https:\/\/6voxehu4w7.execute-api.eu-west-2.amazonaws.com\/SendConversionData$/,
+	'subx-send-conversion-data-test': /^https:\/\/6voxehu4w7.execute-api.eu-west-2.amazonaws.com\/SendConversionData$/,
```
[the remark this answers is not in the document]
Comment (rowanmanning) [reply lib/metrics/services.js:311]: Oh nice I was wondering when we'd get one of these 😁

[the remark this answers is not in the document]
Comment (manasiSantFT) [reply lib/metrics/services.js:311]: Haha. I think escaping a . before amazonaws.com would fix it.

File: lib/metrics/services.js [hunk 2]
```diff
@@ -308,7 +308,8 @@
 	'subs-graphql-test': /^https:\/\/(beta-)?api-t\.ft\.com\/subs\/query(\/graphql)?(\/)?$/,
 	'subs-graphql-api': /^https:\/\/(beta-)?api\.ft\.com\/subs\/query\/api\//,
 	'subs-graphql-api-test': /^https:\/\/(beta-)?api-t\.ft\.com\/subs\/query\/api\//,
-	'subx-send-conversion-data': /^https:\/\/6voxehu4w7.execute-api.eu-west-2.amazonaws.com\/SendConversionData$/,
+	'subx-send-conversion-data-test': /^https:\/\/6voxehu4w7.execute-api.eu-west-2\.amazonaws.com\/SendConversionData$/,
```
[the remark this answers is not in the document]
Comment (manasiSantFT) [reply lib/metrics/services.js:311]: Escaping a `.` is not helping. @rowanmanning  any inputs here?

[the remark this answers is not in the document]
Comment (rowanmanning) [reply lib/metrics/services.js:311]: It'll need to be all the dots to make it stop complaining I think

[the remark this answers is not in the document]
Comment (manasiSantFT) [reply lib/metrics/services.js:311]: Let me try that

[the remark this answers is not in the document]
Comment (manasiSantFT) [reply lib/metrics/services.js]: It worked. Merging this. Thank you

**Pull request merged**